internal/namespaces/k8s/v1beta4/custom_node.go

```diff
@@ -0,0 +1,57 @@
+package k8s
+
+import (
+	"context"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-cli/internal/human"
+	k8s "github.com/scaleway/scaleway-sdk-go/api/k8s/v1beta4"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	nodeActionTimeout = 10 * time.Minute
+)
+
+//
+// Marshalers
```

Comment:
not everything under is to do with marshalers 

add other comments, or remove this one